sprints/dashboard/helpers.py

```diff
@@ -0,0 +1,276 @@
+import re
+from contextlib import contextmanager
+from typing import ContextManager, Dict, List
+
+from jira import JIRA
+# noinspection PyProtectedMember
+from jira.resources import Board, GreenHopperResource, Issue, Sprint, User as JiraUser
+
+from config.settings.base import JIRA_CUSTOM_FIELDS, JIRA_PASSWORD, JIRA_REQUIRED_FIELDS, JIRA_SERVER, \
+    JIRA_SPRINT_BOARD_PREFIX, JIRA_USERNAME, SPRINT_DEFAULT_COMMITMENT, SPRINT_EPIC_DIRECTIVE, \
+    SPRINT_HOURS_RESERVED_FOR_MEETINGS, SPRINT_RECURRING_DIRECTIVE, SPRINT_REVIEW_DIRECTIVE, \
+    SPRINT_STATUS_EPIC_IN_PROGRESS, SPRINT_STATUS_EXTERNAL_REVIEW, SPRINT_STATUS_MERGED, SPRINT_STATUS_RECURRING, \
+    SPRINT_STATUS_UNFINISHED
```

Comment:
Nit: please format like

```python
from config.settings.base import (
    JIRA_CUSTOM_FIELDS,
    JIRA_PASSWORD,
    JIRA_REQUIRED_FIELDS,
    ...,
)
```
(trailing comma on every line)

This format produces much nicer diffs for future changes and is required by edX's python coding standards, which we generally use as well.